src/sentry/event_manager.py

```diff
@@ -109,6 +109,11 @@
 # Timeout for cached group crash report counts
 CRASH_REPORT_TIMEOUT = 24 * 3600  # one day
 
+issue_rate_limiter = RedisSlidingWindowRateLimiter(
+    **settings.SENTRY_PERFORMANCE_ISSUES_RATE_LIMITER_OPTIONS
+)
+PERFORMANCE_ISSUE_QUOTA = Quota(3600, 60, 60)
```

Comment:
This means 60 issues per hour, with a sliding window granularity of a minute. just dummy values, not sure what we'll really use